### PR TITLE
Add Globus user guide

### LINKED ISSUE #82: Globus Documentation
We've been happily tinkering with Globus for about 6 months now.  Things have gone swimmingly -- we should have some minimal pointers on how to use it (and/or work with @JoshKarpel and Infra to come up with a punchlist before we do so).

At a minimum, we need to document (a) it exists, (b) the endpoint name (and point out that the directory structure allows copies to/from staging), and (c) link to the upstream Globus docs.  I don't feel compelled to spend a lot of time teaching people how to use the service -- I'm OK with assuming that people come knowing how to use Globus and just want to use our instance.

### PR DESCRIPTION
**Pull request by JoshKarpel**
This PR will resolve #82 

To do:
- [x] Instructions for connecting to *our* Globus endpoint (once we have one)
- [ ] Pictures of what things look like on the Globus web interface?
- [ ] Point to https://github.com/JoshKarpel/globus-transfer for file transfer as part of a workflow

### CODE REVIEW

**Comment by JoshKarpel** [comment]
Ready for review... it's feeling kind of minimal, and I wouldn't be opposed to adding more information (what information would be helpful?). What do you think about adding pictures of the Globus website, pointing out the various things I'm trying to describe? Easy to go out of date, but maybe helpful....

File: globus.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+---
+
+[Globus](https://www.globus.org/) is a data management service that lets you
+move files files between **endpoints**, computers that are connected to the
+Globus file transfer network.
+CHTC maintains a Globus endpoint that can serve as either the
+source or destination of a Globus file transfer
+(i.e., you can move your files out of CHTC to some other Globus endpoint,
+or from some other Globus endpoint to CHTC).
+
```
Comment (ChristinaLK) [inline globus.md:14]: This is where I would usually include the doc thesis: 
"this guide will show you how to transfer files to and from CHTC using our Globus endpoint." 

And then maybe put the following paragraph under its own heading, like "Requirements" or "Prerequisites"

File: globus.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+All file transfer via Globus at CHTC is based out of the
+[`/staging`](/file-avail-largedata.shtml) directory.
+Therefore, to use Globus, you will need access to `/staging` 
+(instructions for requesting access are 
+[here](/file-avail-largedata.shtml#1-policies-and-intended-use)).
+
+
+## Using the CHTC Globus Endpoint
+
```
Comment (ChristinaLK) [inline globus.md:23]: What I'm missing here is "to transfer files using the CHTC globus endpoint, you can use the Globus web interface." 

Do people need to create an account on globus to use the web UI? If so, that should maybe be here or above under "requirements"

Comment (ChristinaLK) [reply globus.md:23]: Expanding the first comment above - I think you need an expository paragraph here describing: 
1) you'll be using the web interface
2) in the web interface, you'll select two end points (one will be CHTC)
3) choose files at one endpoint and transfer them to a folder on the other

And then continue with what you already have on how to do that.

File: globus.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+[Globus](https://www.globus.org/) is a data management service that lets you
+move files files between **endpoints**, computers that are connected to the
+Globus file transfer network.
+CHTC maintains a Globus endpoint that can serve as either the
+source or destination of a Globus file transfer
+(i.e., you can move your files out of CHTC to some other Globus endpoint,
+or from some other Globus endpoint to CHTC).
+
+All file transfer via Globus at CHTC is based out of the
+[`/staging`](/file-avail-largedata.shtml) directory.
```
Comment (ChristinaLK) [inline globus.md:16]: also the transfer server, yes?

File: globus.md (hunk 1 above)
Comment (ChristinaLK) [reply globus.md:14]: Also - somewhere you should talk about when/why you would use Globus (big file transfers, coming from a Globus endpoint, etc.). Maybe in the intro here?

File: globus.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+We only recommend using Globus Connect Personal if you are also working with
+some other Globus endpoint (not just CHTC and your computer).
+If you are just moving files between CHTC and your own computer, traditional
+file transfer tools like `rsync` will likely be more efficient.
+
+
+## Non-Interactive File Transfer
+
+As the scale of your research increases, you will probably want to move away
```
Comment (ChristinaLK) [inline globus.md:79]: Definitely need more details here! Either tell people that they need to read the README for the linked repo (which we should also review before merging this PR) or include more details in this guide.

Comment (JoshKarpel) [reply globus.md:79]: I think I'm going to cut this for now - I haven't touched that project for a while and want to do some work on it. I'll do a second PR to discuss it once I'm more comfortable with its state.

File: globus.md (hunk 3 above)
Comment (JoshKarpel) [reply globus.md]: Yep; so noted.

File: globus.md (hunk 4 above)
Comment (JoshKarpel) [reply globus.md]: I did indeed cut this for now.

File: globus.md (hunk 2 above)
Comment (JoshKarpel) [reply globus.md:37]: Added the expository paragraph, and notes about how to log in to the Globus web interface.

File: globus.md (hunk 1 above)
Comment (JoshKarpel) [reply globus.md:21]: I made a Prerequisites section, and reworked the introduction to note where Globus is most useful (primary when someone else wants to you to use Globus, for whatever reason).

**Comment by JoshKarpel** [comment]
@jmvera255 would you mind running through the actual steps, if you've got time? If not, I'm ok with merging now and fixing it when someone runs into problems trying to use it :)

**Comment by jmvera255** [comment]
@JoshKarpel I think I can squeeze in a test by the end of the week

**Comment by ChristinaLK** [comment]
I can also try to test for real provided I can get an endpoint running on my computer.

**Comment by JoshKarpel** [comment]
@ChristinaLK @jmvera255 good for merge?

File: globus.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+[Globus](https://www.globus.org/) is a data management service that lets you
+move files files between **endpoints**, computers that are connected to the
+Globus file transfer network.
+Globus is primarily useful when you need to move large amounts of data to or 
+from somewhere that is already providing a Globus endpoint.
+For example, a collaborator might provide shared data through Globus,
+or might expect you to send data to them through Globus.
+
+This guide will show you how to execute such transfers using the CHTC
```
Comment (ChristinaLK) [inline globus.md:15]:
```suggestion
This guide will show you how to execute such transfers to and from CHTC using the CHTC
```

File: globus.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+You can use the Globus web interface to transfer files to and from CHTC.
+In the web interface, you can select two endpoints and then initiate a transfer
+between them.
+
+The first step is to find the CHTC Globus endpoint.
+The CHTC Globus endpoint's name is `chtc#staging`.
+It can be found in the Globus web interface 
+[here](https://app.globus.org/file-manager/collections/d0bae6da-db3b-11ea-85a2-0e1702b77d41/overview?back=endpoints),
+or by searching endpoints for that name.
+If you  need the actual endpoint UUID, it is listed on that page near the bottom
```
Comment (ChristinaLK) [inline globus.md:47]:
```suggestion
If you  need the actual endpoint UUID, it is listed on the above page near the bottom
```

**Pull request merged**